src/pages/discord/link/components/card/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { BlurBackground } from "~/components/layout/BlurBackground";
+import { Container } from "~/components/layout/Container";
+import { Flex } from "~/components/layout/Flex";
+import { Logo } from "~/components/layout/Header";
+import { Wallet } from "~/components/Wallet";
+
+const Card = () => {
```

Comment:
Usually I name this type of component `View` and for not-next-page components we always use named exports like `export const View = ...`